modules/distribution/src/repository/resources/dbscripts/apimgt/mysql.sql

```diff
@@ -317,61 +319,61 @@ CREATE TABLE IF NOT EXISTS SP_PROVISIONING_CONNECTOR (
 ALTER TABLE SP_PROVISIONING_CONNECTOR ADD CONSTRAINT PRO_CONNECTOR_APPID_CONSTRAINT FOREIGN KEY (APP_ID) REFERENCES SP_APP (ID) ON DELETE CASCADE;
 
 CREATE TABLE IF NOT EXISTS IDP (
-                  ID INTEGER AUTO_INCREMENT,
-                  TENANT_ID INTEGER,
-                  NAME VARCHAR(254) NOT NULL,
-                  IS_ENABLED CHAR(1) NOT NULL DEFAULT '1',
-                  IS_PRIMARY CHAR(1) NOT NULL DEFAULT '0',
-                  HOME_REALM_ID VARCHAR(254),
-                  IMAGE MEDIUMBLOB,
-                  CERTIFICATE BLOB,
-                  ALIAS VARCHAR(254),
-                  INBOUND_PROV_ENABLED CHAR (1) NOT NULL DEFAULT '0',
-                  INBOUND_PROV_USER_STORE_ID VARCHAR(254),
-                  USER_CLAIM_URI VARCHAR(254),
-                  ROLE_CLAIM_URI VARCHAR(254),
-                  DESCRIPTION VARCHAR (1024),
-                  DEFAULT_AUTHENTICATOR_NAME VARCHAR(254),
-                  DEFAULT_PRO_CONNECTOR_NAME VARCHAR(254),
-                  PROVISIONING_ROLE VARCHAR(128),
-                  IS_FEDERATION_HUB CHAR(1) NOT NULL DEFAULT '0',
-                  IS_LOCAL_CLAIM_DIALECT CHAR(1) NOT NULL DEFAULT '0',
+			ID INTEGER AUTO_INCREMENT,
+			TENANT_ID INTEGER,
+			NAME VARCHAR(254) NOT NULL,
+			IS_ENABLED CHAR(1) NOT NULL DEFAULT '1',
+			IS_PRIMARY CHAR(1) NOT NULL DEFAULT '0',
+			HOME_REALM_ID VARCHAR(254),
+			IMAGE MEDIUMBLOB,
+			CERTIFICATE BLOB,
+			ALIAS VARCHAR(254),
+			INBOUND_PROV_ENABLED CHAR (1) NOT NULL DEFAULT '0',
+			INBOUND_PROV_USER_STORE_ID VARCHAR(254),
+ 			USER_CLAIM_URI VARCHAR(254),
+ 			ROLE_CLAIM_URI VARCHAR(254),
+  			DESCRIPTION VARCHAR (1024),
+ 			DEFAULT_AUTHENTICATOR_NAME VARCHAR(254),
+ 			DEFAULT_PRO_CONNECTOR_NAME VARCHAR(254),
+ 			PROVISIONING_ROLE VARCHAR(128),
+ 			IS_FEDERATION_HUB CHAR(1) NOT NULL DEFAULT '0',
+ 			IS_LOCAL_CLAIM_DIALECT CHAR(1) NOT NULL DEFAULT '0',
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04